airbyte-integrations/connectors/source-awtomic/integration_tests/sample_state.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
If you're not using the file, remove it. Are all streams full refresh only? If some are incremenental, pleae add test states.